aspnet/ComboBox/ComboBox-Dependency.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Dependencies of the ComboBox control for Syncfusion ASP.NET WebForm
-description: Describes about dependencies of the ComboBox control for Syncfusion ASP.NET WebForm 
+title: Dependencies of ASP.NET WebForm ComboBox control | Syncfusion
```

Comment:
![image](https://github.com/syncfusion-content/asp.netwebforms-docs/assets/95272441/f95fdcaa-d549-4bf6-9f63-13aee31d5977)
Dependencies of ASP.NET WebForm ComboBox Control | Syncfusion

Reply:
renamed